tables/pu-in-g1.utb

```diff
@@ -24,4 +24,19 @@
 # To report any bugs or any suggestion, please write to d@saksham.org and sreeja.param@gmail.com
 
```

Comment:
This text in the table header should probably also be changed (similar to gurumuki.cti) to document your contribution.